src/accordion/styled.ts

```diff
@@ -63,3 +63,15 @@ export const StyledPanel = styled.div<
   ${getStylePreset('accordion.panel', 'panel')};
   ${logicalProps('accordion.panel', 'panel')}
 `;
+
+export const StyledAccordionWrapper = styled.div<
+  Pick<AccordionProps, 'overrides' | 'expanded'>
+>`
+  ${logicalProps('')}
```

Comment:
Why is not having a default path?
```suggestion
  ${logicalProps('accordion')}
```